src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -1,36 +1,136 @@
 @import 'extra-actions';
 
-.ez-btn {
-    display: inline-block;
-    padding: calculateRem(11px) calculateRem(16px);
+.ibexa-btn {
+    display: inline-flex; // TODO: can we use flex in buttons???
+    padding: calculateRem(16px);
     outline: none;
     border: none;
     border-radius: $ibexa-border-radius;
     text-decoration: none;
-    font-size: calculateRem(15px);
+    font-size: calculateRem(14px);
     line-height: calculateRem(16px);
-    transition: all 0.2s $ibexa-admin-transition;
+    transition: all 0.2s $ibexa-admin-transition, border-width 0s, padding 0s;
 
-    &:hover,
-    &:focus,
-    &:active {
-        border: none;
-        outline: none;
-        text-decoration: none;
+    $btn-focus-box-shadow: 0 0 0 3px rgba($ibexa-color-primary, 0.2);
+
+    &:focus {
+        box-shadow: $btn-focus-box-shadow;
     }
 
     &--primary {
-        color: $ibexa-white;
-        background-color: $ibexa-color-primary;
+        color: $ibexa-color-white;
+        background-image: linear-gradient(to right, $ibexa-color-danger 0%, $ibexa-color-primary 100%);
```

Comment:
Will this gradient be only in the buttons? Because if not, then maybe let's use a variable?